### ISSUE TITLE
Learn from Mike Shah video on OCaml

### ISSUE THREAD

**Issue by cuihtlauac** [ocaml/ocaml.org#2026]
This is derived from @MikeShah YouTube Video [Episode 8 - First Impression - ocaml](https://youtu.be/Aa5b6Y8NvGQ)

* Make it obvious place to start is `utop` command: applies to install instructions (http://ocaml.org/install and [Installing OCaml](https://ocaml.org/docs/installing-ocaml)) and [A Tour of OCaml](https://ocaml.org/docs/tour-of-ocaml).
* Add the nice `10 :: u` example (at around 17:00)
* Bring up explanation on single equality symbol double meaning earlier in the text
* Show `#help;;`
* Add a link to generated assembly. Some enjoy that from the beginning.
* Use `cat "cat" "dog"` instead of `cat "ha" "ha"`, nicer and no repetition

**Comment by sabine** [comment]
I don't think we can tell people to use Utop in the install instructions, but we can mention that Utop is a REPL.

Tour of OCaml, yes, maybe we can shorten the intro to make it really stand out that people should use utop.

**Comment by cuihtlauac** [comment]
> I don't think we can tell people to use Utop in the install instructions, but we can mention that Utop is a REPL.

In https://ocaml.org/docs/installing-ocaml we already conclude by launching `utop`, as a sanity check. I suggest we duplicate this in https://ocaml.org/install for the same purpose.

> Tour of OCaml, yes, maybe we can shorten the intro to make it really stand out that people should use utop.

We used to have something. We've deleted when we created https://ocaml.org/docs/toplevel-introduction. This is fine, but it is one click too far for some.
